crown-court-proceeding/build.gradle

```diff
@@ -46,6 +49,12 @@ dependencies {
     implementation "io.sentry:sentry-logback"
     implementation "io.sentry:sentry-spring-boot-starter"
 
+    implementation group: 'org.springframework.cloud', name: 'spring-cloud-starter-aws', version: '2.2.6.RELEASE'
+    implementation group: 'org.springframework.cloud', name: 'spring-cloud-starter-aws-messaging', version: '2.2.6.RELEASE'
```

Comment:
Should we define a constant for the version and use it here?

Reply:
Updated